pkgs/data/icons/super-tiny-icons/default.nix

```diff
@@ -0,0 +1,46 @@
+{ pkgs, lib, stdenvNoCC, fetchFromGitHub, }:
+stdenvNoCC.mkDerivation rec {
+  pname = "super-tiny-icons";
+  version = "20221107-${lib.strings.substring 0 7 rev}";
+  rev = "b4d5a3be04c99ec0a309ac9e0d0b21207c237c7d";
+
+  src = fetchFromGitHub {
+    owner = "edent";
+    repo = "SuperTinyIcons";
+    inherit rev;
+    sha256 = "670ndAuBrZBr7YOTQm2zHJfpiBC56aPk+ZKMneREEoI=";
+  };
+
+  outputs = [ "out" "doc" ];
+  phases = [ "installPhase" ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/icons/SuperTinyIcons
+    for icondir in $(find $src/images -type d); do
+      cp -r $icondir $out/share/icons/SuperTinyIcons/
+    done
+
+    install -Dm644 -t $doc/share/doc/SuperTinyIcons/ \
+      $src/CODE_OF_CONDUCT.md \
+      $src/CONTRIBUTING.md \
+      $src/ISSUE_TEMPLATE.md \
+      $src/LICENSE \
+      $src/README.md
+
```

Comment:
We normally do not copy all this stuff because NixOS has no easy way to browse them all.